Qausim/react-weather-app
# Implement Loading Indicator

#### What does this PR do?
Implements a loading indicator that displays when API request starts

#### Description of Task to be completed?
Setup a component for the loading indicator

#### How should this be manually tested?
Clone this repo, cd into it
- Run `npm start`, visit `localhost:8080`, enter a location to search, the loading indicator should display

#### Any background context you want to provide?
There should be an indicator to depict the app is busy to the user while API request persists
